lib/src/home/repos/home_view_repo.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:tech_tuesday/core/models/article_model.dart';
+import 'package:tech_tuesday/core/network/api.dart';
+
+class HomeViewRepo {
```

Comment:
this file also covers API i have little knowledge here so i would say i don't quiet understand it